src/ros_vision_interaction/src/vision-interaction/reading_evaluator/__init__.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3.8
```

Comment:
It might be good to test this file if you can think of an easy way to do so.

Reply:
I haven't been able to find an easy way to test it, but I'll keep this on the todo list.